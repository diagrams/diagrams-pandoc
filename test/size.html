<p>Here is a square:</p>
<img src="image.png" />