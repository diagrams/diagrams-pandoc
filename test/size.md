Here is a square:

``` {.diagram width=100 height=100}
example = square 1
```

