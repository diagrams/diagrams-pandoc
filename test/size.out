[ Para
    [ Str "Here"
    , Space
    , Str "is"
    , Space
    , Str "a"
    , Space
    , Str "square:"
    ]
, Para
    [ Image
        ( "" , [] , [] ) [] ( "images/4bcef48f2a2f113a.png" , "" )
    ]
]